File: Minimum-Operations-to-Exceed-Threshold-Value-I.cpp
class Solution {
public:
    int minOperations(vector<int>& nums, int k) {
        int count = 0;
        sort(nums.begin(), nums.end());
        for(auto num : nums) {
            if(num < k) {
                count++;
            }
        }
        return count;
    }
};

File: Minimum-Operations-to-Exceed-Threshold-Value-I.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var minOperations = function (nums, k) {
    let count = 0;
    nums.sort((a, b) => a < b);
    for (const num of nums) {
        if (num < k) {
            count++;
        }
    }
    return count;
};